src/SFA.DAS.RoATPService.Data/OrganisationRepository.cs

```diff
@@ -285,5 +287,29 @@ public async Task<OrganisationRegisterStatus> GetOrganisationRegisterStatus(stri
 
             }
         }
+
+        public async Task<IEnumerable<Engagement>> GetEngagements()
+        {
+            var connectionString = _configuration.SqlConnectionString;
+
+            using (var connection = new SqlConnection(connectionString))
+
+            {
+                if (connection.State != ConnectionState.Open)
+                    await connection.OpenAsync();
+
+                var sql = $@"select ID, ProviderId, CreatedOn,
+                                Case OrganisationStatusId
+                                When 0 then 'REMOVED'
```

Comment:
Just wondering if these values should be added to the OrganisationStatus lookup table? That way if we add new statuses in the future, we don't forget to update this query

Reply:
Talked to Paul about this, and we'll go with what you said.  However, as it's going to change orgStatus table, we want this to get into master ASAP to stop the database update nonsense you sometimes get.  I'll ping you once the changes are in.